docs/app/pages/components/components-sections/wizard/wizard/wizard.component.html

```diff
@@ -78,6 +78,9 @@
     <tr uxd-api-property name="finishAriaLabel" type="string">
         Defines the text for the aria label on the 'Finish' button.
     </tr>
+    <tr uxd-api-property name="disableNextWhenInvalid" type="boolean">
```

Comment:
I could be wrong but looking at the code adding this to the wizard/marquee wizard component doesn't look like it does anything? It looks like it only takes into account this value when it is applied to the step element:

```typescript
isNextDisabled(): boolean {
    const { valid, disableNextWhenInvalid } = this.getCurrentStep();
    return disableNextWhenInvalid && !valid;
}
```

So we should either remove it from the wizard and marquee wizard component and only have it on the step component, or support both, but we would need to make it works when used on the wizard components and we would need to add tests. @donna-marijne do you think it is beneficial to have on both a wizard and step component?

Reply:
Yes, I suggested a two-tier approach above, citing the use case of a wizard which contains only one step requiring async validation (i.e. when you hit next it makes an async call to update the valid status). This should not be too complicated to implement, but if you don't think it's worthwhile we can just define it at the step level. The issue with that is potentially repetitive configuration since the default is false (for backward compatibility reasons), but for someone who wants to use this feature they probably want it enabled on either all or most of the wizard steps.

Reply:
That sounds fine to me, if we could then update the code to support both step and wizard level options (step level taking precedence) and also add a test to cover wizard level and a test to cover that step level overrides the wizard level

Reply:
So something like:
```ts
disableNextWhenInvalid: boolean | undefined = undefined;
```

And then
```ts
return (step.disableNextWhenInvalid === true || this.disableNextWhenInvalid) && !valid;
```

I'm just freehand typing this but hopefully you get the idea?

Reply:
Actually I guess it would be more like
```ts
return (step.disableNextWhenInvalid === undefined ? this.disableNextWhenInvalid : step.disableNextWhenInvalid) && !valid;
```